include/view.h

```diff
@@ -127,7 +135,8 @@ struct view {
 	bool ssd_titlebar_hidden;
 	enum ssd_preference ssd_preference;
 	bool minimized;
-	bool maximized;
+	bool hmaximized; /* horizontally maximized */
+	bool vmaximized; /* vertically maximized */
```

Comment:
Just a random thought: maybe we could change `view->maximized` to `enum view_axis` and add a `VIEW_AXIS_NONE` as `0`? Then `if (!view->maximized)` would still work. Didn't completely think this through though.

Reply:
I like this. We could also allow a `direction` of `none` in rc.xml (different from Openbox, but that's fine).

I would suggest that we still explicitly check `view->maximized != VIEW_AXIS_NONE` or `view->maximized == VIEW_AXIS_BOTH` though (depending on the desired semantics).